.changeset/many-rivers-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Add new FeedbackButton for comet/admin
```

Comment:
This changeset is already for `comet/admin` so it doesn't need to be explicitly mentioned

```suggestion
Add `FeedbackButton` component
```